test/e2e/storage/external/testdata/example.yaml

```diff
@@ -0,0 +1,19 @@
+StorageClass:
+  FromExistingClassName: example
+DriverInfo:
+  Name: example
+  RequiredAccessModes:
+  - ReadWriteOnce
+  Capabilities:
+    persistence: true
+    multipods: true
+    exec: true
+    block: true
+    fsGroup: true
+    topology: true
+    controllerExpansion: true
+    nodeExpansion: true
+    volumeLimits: false
```

Comment:
Can you include `snapshotDataSource: true`?

Reply:
Done